From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nitin Yewale <nyewale@redhat.com>
Date: Thu, 12 Jan 2023 14:28:49 -0600
Subject: [PATCH] libmultipath: remove pathgroup wildcard options

The multipathd command "multipathd show wildcards" shows the pathgroup
format wildcards, but there is no way to use them in a multipathd
command.

Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 libmultipath/print.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/libmultipath/print.c b/libmultipath/print.c
index 082e4e30..a6e4c774 100644
--- a/libmultipath/print.c
+++ b/libmultipath/print.c
@@ -803,13 +803,6 @@ int snprint_wildcards(struct strbuf *buff)
 				       pd[i].wildcard, pd[i].header)) < 0)
 			return rc;
 
-	if ((rc = append_strbuf_str(buff, "\npathgroup format wildcards:\n")) < 0)
-		return rc;
-	for (i = 0; pgd[i].header; i++)
-		if ((rc = print_strbuf(buff, "%%%c  %s\n",
-				       pgd[i].wildcard, pgd[i].header)) < 0)
-			return rc;
-
 	return get_strbuf_len(buff) - initial_len;
 }
 
